ocs_ci/deployment/deployment.py

```diff
@@ -660,6 +661,12 @@ def deploy_ocs_via_operator(self, image=None):
                     "enable": True,
                 }
 
+        if config.DEPLOYMENT["ceph_debug"]:
+            setup_ceph_debug()
+            cluster_data["spec"]["managedResources"] = {
+                "cephConfig": {"reconcileStrategy": "ignore"}
```

Comment:
What does this do?

Reply:
it's for enabling https://github.com/openshift/ocs-operator/pull/1091 for having the new Ceph configuration be persistent

Reply:
If it's a workaround for BZ 1928471, it should be placed in a separate pull request with explanation as noted in [Tracking of workarounds](https://ocs-ci.readthedocs.io/en/latest/workarounds.html). This helps to make sure our testing environment is valid in a long term.

Reply:
This PR is for allowing running with Ceph in debug, to allow us to deploy OCS in that way, mainly for investigation purposes.
Running with Ceph in debug is needed for example, for the investigation of BZ 1928471. It's not a workaround for this bug.

Reply:
Ah, ok.